staging/src/k8s.io/client-go/tools/cache/reflector.go

```diff
@@ -80,7 +80,11 @@ type Reflector struct {
 	// lastSyncResourceVersionMutex guards read/write access to lastSyncResourceVersion
 	lastSyncResourceVersionMutex sync.RWMutex
 	// WatchListPageSize is the requested chunk size of initial and resync watch lists.
-	// Defaults to pager.PageSize.
+	// If unset, for consistent reads (RV="") it will default to pager.PageSize,
```

Comment:
```suggestion
	// If unset, for consistent reads (RV="") or reads that opt-into arbitrarily old data (RV="0") it will default to pager.PageSize,
```